services/amo/amo.service.js

```diff
@@ -157,6 +157,10 @@ class AmoUsers extends LegacyService {
 //
 // Do not base new services on this code.
 class Amo extends LegacyService {
+  static get category() {
+    return 'other'
+  }
```

Comment:
These are meaningless categories; however these are all legacy services and the goal is to split them up.